src/main/java/org/sagebionetworks/web/client/widget/team/JoinTeamWidget.java

```diff
@@ -164,9 +212,8 @@ public void onFailure(Throwable caught) {
 	}
 	
 	@Override
-	public void sendJoinRequest(String message, boolean isAcceptingInvite) {
+	public void sendJoinRequest(String message) {
 		this.message = message;
-		this.isAcceptingInvite = isAcceptingInvite;
```

Comment:
![image](https://cloud.githubusercontent.com/assets/8346437/9452938/12c70204-4a6c-11e5-8ec7-2e9bc38aeb22.png)

sagetesting3 was invited to the team, presented with a Join button, then when I clicked Accept, the page refreshed, and this was the appearance:
![image](https://cloud.githubusercontent.com/assets/8346437/9452951/23445370-4a6c-11e5-8406-7c7b36ad0437.png)